src/screens/DonateScreen/SegmentedControl.js

```diff
@@ -69,7 +69,7 @@ const styles = StyleSheet.create({
     borderColor: mediumGreyColor,
     borderTopWidth: 1,
     borderBottomWidth: 1,
-    height: 50 * PixelRatio.getFontScale(),
+    height: 50 * Math.min(1.22, PixelRatio.getFontScale()),
```

Comment:
`1.22`? Keen to understand where this numbers came from. If this is intentional, I wonder if there is a more meaningful way for us to calculate this. I.e. some relation to `lineHeight`? or `fontSize`?

Reply:
Yes, that's intentional. But we definitely need a way to make this understandable.

In this case this style is for the button, which is an `h2`. The `h2` has a default font size of 18 and a max size of 22.

We want the height to cap at the same scale. Result: 22 / 18 = 1.22